### DISCUSSION TITLE
Tooling options

### DISCUSSION THREAD

**Discussion by ebgoldstein** [Doodleverse/s2m_engine#2] [Ideas]
gdal
rasterio
rioxarray
buteo https://casperfibaek.github.io/buteo/

**Comment by jmdelvecchio** [comment]
Using netcdfs and `xarray` for self-describing files for when you have N dimension stacks with satellite (or other gridded) data https://learning.nceas.ucsb.edu/2022-09-arctic/sections/08-data-structures-netcdf.html
